server/services/additionalSupportService.ts

```diff
@@ -0,0 +1,15 @@
+import { HmppsAuthClient, OrchestrationApiClient, RestClientBuilder } from '../data'
+import { SupportType } from '../data/orchestrationApiTypes'
+
+export default class AdditionalSupportService {
+  constructor(
+    private readonly orchestrationApiClientFactory: RestClientBuilder<OrchestrationApiClient>,
+    private readonly hmppsAuthClient: HmppsAuthClient,
+  ) {}
+
+  async getAvailableSupportOptions(username: string): Promise<SupportType[]> {
```

Comment:
Opportunity to add some caching here (as we do for list of all prisons) because available support options list very static. Currently we're requesting it each time and storing in user's session.